Src/Support/Google.Apis.Auth.Tests/OAuth2/LimitedLocalhostHttpServerTests.cs

```diff
@@ -29,10 +29,13 @@ namespace Google.Apis.Auth.Tests.OAuth2
 {
     public class LimitedLocalhostHttpServerTests
     {
-        private LocalServerCodeReceiver.LimitedLocalhostHttpServer StartServer()
+        private const string CustomResponseHtml = "<html><body>Custom response!</body</html>";
+
+        private LocalServerCodeReceiver.LimitedLocalhostHttpServer StartServer(bool useCustomResponseHtml = false)
         {
             var url = string.Format(LocalServerCodeReceiver.CallbackUriTemplate127001, 0);
-            return LocalServerCodeReceiver.LimitedLocalhostHttpServer.Start(url, LocalServerCodeReceiver.DefaultClosePageResponse);
+            var responseHtml = useCustomResponseHtml ? CustomResponseHtml : LocalServerCodeReceiver.DefaultClosePageResponse;
+            return LocalServerCodeReceiver.LimitedLocalhostHttpServer.Start(url, responseHtml);
```

Comment:
Hmm. It's not clear how much this is really testing, given that we're giving it the HTML either way. We're not testing the *user* path of either specifying HTML or not specifying it. Is that easy to test? (Basically I'd like to test the two overloads of `LocalServerCodeReceiver` separately.)

Reply:
You're correct of course ;) Updated to test the `LocalServerCodeRecevier` ctors.

Reply:
Hmm. It *looks* to me like we had a breaking change - before #1380, there was a `public static LimitedLocalhostHttpServer Start(string url)` method; now there's only the one with the HTML.
Maybe now's the time to add the old overload back in, and call the appropriate overload from this test code.

Reply:
Ah, I see - it's an internal class. Worries about breaking changes over... but what calls the LimitedLocalhostHttpServer.Start method?

Reply:
Okay, I think I'm with it now - but it feels like if the test is really "on all platforms, going through the public API to use default HTML should result in default HTML". That would use `LocalServerCodeReceiver` directly in both cases. Testing `LimitedLocalhostHttpServer` doesn't feel as useful - and in particular, if you *do* want tests specifically for that class but we're not using it in .NET Framework, let's only run the tests in .NET Core.

Reply:
`LimitedLocalhostHttpServer` definitely needs testing as it tests that various invalid requests are handled correctly, but I agree it on needs testing in .NET Core, as it's not used in full framework. Updated the test to on test on Core.
It's not possible to completely test `LocalServerCodeReceiver` in a unit test, as it starts up a browser for user interaction; this is tested in various integration tests.